gaussian_renderer/__init__.py

```diff
@@ -766,9 +1027,168 @@ def render_final(batched_screenspace_pkg, batched_strategies):
             )
         batched_rendered_image.append(rendered_image)
         batched_compute_locally.append(compute_locally)
+        
         if timers is not None:
             timers.stop("forward_render_gaussians")
     utils.check_initial_gpu_memory_usage("after forward_render_gaussians")
 
+    ########## [END] CUDA Rasterization Call ##########
+    return batched_rendered_image, batched_compute_locally
+  
+def gsplat_render_final(batched_screenspace_pkg, batched_strategies, tile_size=16):
+    """
+    Render the scene. 
+    """
+    timers = utils.get_timers()
+    C = len(batched_strategies)
```

Comment:
C is batch size(num of cameras), right? When I first read this, I thought this is batch size which means we should use B (rather than C). However, gsplat uses C for batch size(num of cameras) here. What do you think is better to use here?

Reply:
I guess C is for "cameras" in gsplat and I just follow their style so it's easier to code. But I agree that using B here causes fewer misunderstandings. Will fix it.